PowerPointLabs/Test/FunctionalTest/BaseFunctionalTest.cs

```diff
@@ -40,6 +41,11 @@ public void Setup()
                 CloseActivePpInstance();
             }
 
+            if (!Directory.Exists(PathUtil.GetTempTestFolder()))
+            {
+                Directory.CreateDirectory(PathUtil.GetTempTestFolder());
```

Comment:
UT will likely have it soon, because FT and UT shares the same testing framework (those under utils folder) and so has the ability to export to tmp.